### DISCUSSION TITLE
please support input -list file

### DISCUSSION THREAD

**Discussion by hktalent** [projectdiscovery/katana#182] [General]
please support input -list file

Parameters consistent with other tools

-list

-nc

-json -o

thanks

**Comment by hktalent** [comment]
<img width="238" alt="image" src="https://user-images.githubusercontent.com/18223385/202123266-fc8678cc-efd0-483c-8cbd-78a34d6bd16d.png">
For configurations with the same format, it is recommended to uniformly use the same configuration file
@ehsandeep

**Reply by ehsandeep** [reply]
@hktalent `config.yaml` is unique across all the project; other configs are project depended, for example, `form-config` is not same as `provider-config`.

**Comment by ehsandeep** [comment]
@hktalent not sure where you are checking options, but those options are already available in katana.

```console
katana -h input

INPUT:
   -u, -list string[]  target url / list to crawl
```

```console
katana -h output

Flags:
OUTPUT:
   -j, -json           write output in JSONL(ines) format
   -nc, -no-color      disable output content coloring (ANSI escape codes)
   ```

**Reply by hktalent** [reply]
Well, I just found that "target url/list to crawl" does not contain the file keyword, so I thought it did not support file
